WebRoot/js/zimbraMail/briefcase/view/ZmPreviewPaneView.js

```diff
@@ -355,11 +356,10 @@ function() {
 	this._previewView.set(this._delayedSelectionItem);
 };
 
-ZmPreviewPaneView.prototype._listSelectionListener = function(ev, item) {
-
+ZmPreviewPaneView.prototype._listSelectionListener = function (ev, item) {
     var item = item || ev.item;
     if (!item) {
-    	return;
+        return;
```

Comment:
use tab for indentation

Reply:
this file has mix of tabs and spaces, but don't change entire file
only use tabs in your code

Reply:
done.